src/blocks/custom/form/form.php

```diff
@@ -45,8 +53,9 @@ class="<?php echo esc_attr( "{$block_class}__form js-{$block_class}-form" ); ?>"
     action="<?php echo esc_attr( $form_action ); ?>"
     method="<?php echo esc_attr( $form_method ); ?>"
     target="<?php echo esc_attr( $form_target ); ?>"
-    <?php ! empty( $form_id ) ? printf( 'id="%s"', esc_attr( $form_id ) ) : ''; ?>
     data-form-type="<?php echo esc_attr( $form_type ); ?>"
+    <?php ! empty( $form_id ) ? printf( 'id="%s"', esc_attr( $form_id ) ) : ''; ?>
```

Comment:
What is the type of `$form_id`? If it's integer the `%d` should be used. With that, technically, you don't need to escape it when passing to `printf` as an argument (cool trick 😄 ).

Or you can 'escape' it using `(int)` typecasting. Which is a valid way of escaping integers.

Reply:
ID as in `id, class`, not `id` as in post type id 😄  (meaning it's a string, not int)

Reply:
Oh, I see now 👍  Then it's ok 🙂